airbyte-webapp/src/views/Connector/ConnectorForm/components/Property/LabelInfo.module.scss

```diff
@@ -18,6 +18,17 @@
   margin-top: 5px;
 }
 
```

Comment:
It looks like you forked the `airbytehq` repo to create this PR. Our pattern is to use feature branches instead of forks for our PRs, since we have a bunch of CI checks that only run on feature branches and not on forks.

We may actually need to recreate this PR on a feature branch instead of a fork to get those CI checks to run